server/src/main/java/fr/uga/l3miage/tp4/repositories/CandidateRepository.java

```diff
@@ -0,0 +1,16 @@
+package fr.uga.l3miage.tp4.repositories;
+
+import fr.uga.l3miage.tp4.models.CandidateEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDate;
+import java.util.Set;
+
+@Repository
+public interface CandidateRepository extends JpaRepository<CandidateEntity, LocalDate> {
```

Comment:
Ah ben non l'id de CandidatEntity n'est pas de type `LocalDate` mais bien de `Long`
```suggestion
public interface CandidateRepository extends JpaRepository<CandidateEntity,Long> {
```